SweaterBrain/ClientApp/src/components/SweaterBrain.js

```diff
@@ -27,40 +27,34 @@ export class FetchData extends Component {
 
         return (
             <div>
-                <Card
-                >
+                <Card className="col-12 text-center">
                     <CardBody>
                         <CardTitle tag="h5">
-                            Today's sweater recommendation: 
+                            
                         </CardTitle>
                         <CardSubtitle
                             className="mb-2 text-muted"
                             tag="h6"
                         >
-                            Card subtitle
+                            Medium Weight do to a current temperature of  [TEMPERATURE HERE]
                         </CardSubtitle>
                         <img src={require('../images/medium_sweater.png').default} width="20%" height="20%" />
-                        <CardText>
-                            Some quick example text to build on the card title and make up the bulk of the card's content.
-                        </CardText>
-                        <Button>
-                            Button
-                        </Button>
                     </CardBody>
                 </Card>
             </div>
         );
     }
 
   render() {
-    let contents = this.state.loading
-      ? <p><em>Loading...</em></p>
-      : FetchData.renderForecastsTable(this.state.forecasts);
+    let contents = SweaterBrain.renderForecastsTable(this.state.forecasts);

```

Comment:
Elon
Is he Hitler? Is he?


Fix his 'neutral pathways' why does he think we need to be working right now?

Reply:
Fix him, let him relax

Reply:
His "God" is work. You can't fix that???

Reply:
Not even cousin Jimmy?

Reply:
Or Uncle Chris?

Reply:
Or Taze?

Reply:
Or Sissy/Ruth

Reply:
How do I even say goodbye?

Reply:
He talks through his work, so I was hoping I could reach him this way....

Reply:
Berried Alive it is then...

Reply:
No pulse huh

Reply:
deleting now